operations/rotate-linux-windows-passwords/README.md

```diff
@@ -0,0 +1,102 @@
+# Simple Password Rotation with HashiCorp Vault
```

Comment:
I think this guide should be categorized under the `secrets` folder vs `operations` since it's focused on rotating secrets. Some supporting history there below...

- `operations` more oriented towards operationalizing Vault (provisioning, root key rotation, HA, DR, replication, autounseal, etc.)
- `secrets` is for guides where the use case is focused on interacting with the secrets engines, generally not including `transit` or `pki`
- `identity` or soon to `iam` (I think?) is for guides where the use case is focused on interacting with auth backends (secure intro with AppRole, AWS or GCP auth, etc.)
- `encryption` is anything interacting with the `transit` or `pki` secret engines (communication of communication and data in-flight or at rest)

Reply:
Ok, i'll move it to `secrets`